cmd/generate.go

```diff
@@ -71,5 +96,7 @@ func NewGenerateCommand() *cobra.Command {
 		},
 	}
 
+	command.Flags().StringVarP(&configPath, "config-path", "c", "", "path to a configuration file (YAML, JSON, envfile) to use")
+	command.Flags().StringVarP(&secretName, "secret-name", "s", "", "name of a Kubernetes Secret containing Vault configuration data in the argocd namespace of your ArgoCD host")
```

Comment:
Can you add something to make it very clear that `secret-name` will only work if in ArgoCD?

Reply:
I was just looking for something like `(Only available when used in ArgoCD)` to make it very obvious. Someone might read this `name of a Kubernetes Secret containing Vault configuration data in the argocd namespace of your ArgoCD host` and not really understand.